src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    if (robot[key] in result) {
+      return null;
+    }
```

Comment:
better use `hasOwnProperty` to check if key exist in object.

Reply:
Thank you! I'll take note😊